src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Heavy, sans-serif;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Book, sans-serif;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+html {
+  font-size: 16px;
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+}
+
+.search-bar__wrapper {
+  position: relative;
+  margin: 20px 0;
+}
```

Comment:
You have margin collapse this way, use either margin-top or margin-bottom

Reply:
Done.